### PR TITLE
Store/cache in file responses(without checking existing data)

### PR DESCRIPTION
**Pull request by PannoniansDavid**
According to available time, delivered saving responses into .json files to be reviewed. 
I'll continue working on second part of US.

### CODE REVIEW

File: public/media-hunt/.env.example [hunk 1]
```diff
@@ -1 +1 @@
-TMDB_API_TOKEN=
+TMDB_API_TOKEN=eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJmMzEwZDg2NTM4OTkzYmNhMjNlNWUyOWNkYjk3NGFlNCIsInN1YiI6IjVjMWM5OTQwOTI1MTQxMjdmODRlMWVmNSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.csMyKzkblGa8r-2EPryAbFrm_k5LcOZae1tgvTAXxFY
```
Comment (pvukmanovic-pannonians) [inline public/media-hunt/.env.example:1]: Please never save actual values in example here. this should only have "keys", left of the "=" sign.

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]
Let's have just one, single json in our project.
Also, please use "key = value" system to store data. This will just store one single response from movie, configuration, whatever, we need to be able to cache more than one response, hence, key -> value.

Secondly, for all files, please have your check FIRST check the json file if the data exists, if it doesn't, then perform HTTP request.

Upon refactor, let's sync some more.

**Review by pvukmanovic-pannonians — CHANGES_REQUESTED** [review]
Please refer to Nikolina's OBS. :D

**Comment by pvukmanovic-pannonians** [comment]
@PannoniansDavid ti si ovo odradio sve cool, samo par stvari:
1. Treba da se uradi JSON.stringify i JSON.parse prilikom ucitavanja podataka iz json fajla i
2. ucitavanje fajla const `json = "./savedResponse.json";` iz komandi bi manje vise znacilo da bi imao vise JSON fajlova. Sto iskreno i nije lose, cak sta vise, svaki od njih bi onda bio chunkovan i drzao manje podataka, tako da taj pristup mi se cak i svidja
3. onaj append file nece piti vodu u ovom formatu. mozda da si koristio nesto tipa [lowdb](https://github.com/typicode/lowdb), to bi onda bio prihvatljiv pristup, al' nebitno, prosli smo to ranije ^_^

Bottom line, ovo bi radilo s'tim da eto, json parse i stringify funkcije plus ne append file vec samo read u i write u isti fajl
eto. ^_^
